### DISCUSSION TITLE
connection lost after 1 days when upgraded to 1.1.1

### DISCUSSION THREAD

**Discussion by ArrayZoneYour** [Chia-Network/chia-blockchain#2551] [General]
My master mode stop syncing after block which height is 177071, I click the Farmer Puzzle Hash to view the detail on chiaexplorer and found that farmer doesn't have the record on 177071, and he won 25% of the coins recently. Is that possible for a 51% attack?
or a simple bug in front end?

cc @wjblanke @hoffmang9 

![Screenshot from 2021-04-24 14-42-22](https://user-images.githubusercontent.com/21001457/115950068-abc57580-a50b-11eb-9451-6210e2ab9f4a.png)

![Screenshot from 2021-04-24 14-45-51](https://user-images.githubusercontent.com/21001457/115950103-d7486000-a50b-11eb-9e20-e522174e2350.png)

**Comment by ArrayZoneYour** [comment]
How can I make it sync again? Will it resync automatically in 1~2 hours ?

**Reply by ArrayZoneYour** [reply]
Oh, it is my fault. Is block height = commit height + 1~4?

**Reply by ArrayZoneYour** [reply]
I connect to my friends' 8444 node manually and make it sync again.

**Reply by anderzheng** [reply]
could you tell me the ip with the lastest height? I cann't sync at height 175306, and have been 12 hours.

**Reply by ArrayZoneYour** [reply]
@anderzheng You can have a try on 123.14.34.167 ? I am not sure whether you can reuse my connections.

**Reply by anderzheng** [reply]
I opened my 8444 port, and fixed the sync problem, Thank you.
